packages/platforms/react-native/lib/retry-queue/index.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  type Delivery,
+  type RetryQueue,
+  type RetryQueueFactory
+} from '@bugsnag/core-performance'
+
+import RetryQueueDirectory, { type MinimalFileSystem } from './directory'
+import FileBasedRetryQueue from './file-based'
+import { PERSISTENCE_DIRECTORY } from '../persistence'
+
+export default function createRetryQueueFactory (fileSystem: MinimalFileSystem): RetryQueueFactory {
+  return function fileBasedQueueFactory (delivery: Delivery, _retryQueueMaxSize: number): RetryQueue {
```

Comment:
unused `_retryQueueMaxSize` parameter

Reply:
That's intentional since the `RetryQueueFactory` type has this parameter:

https://github.com/bugsnag/bugsnag-js-performance/blob/0448e7a43823bb49b555eee48bcb497f7e5d203f/packages/core/lib/retry-queue.ts#L8